components/scream/cime_config/buildnml

```diff
@@ -185,16 +196,19 @@ def buildnml(case, caseroot, compname):
         if not os.path.isdir(target):
             os.mkdir(target)
 
-        for item in os.listdir(src):
+        for gitem in [os.path.basename(yaml_tgt), "*.ascii"]:
```

Comment:
What files are we processing here? Are these the vertical coordinate files (like `scream-128m.ascii')? Or are there others? If it's the vcoord files, we should prob init them differently.

@AaronDonahue I think EAM puts hyai/hyam/hybi/hybm in the nc initial conditions file, right? Perhaps we should do the same, so that we do not need to provide the vertical coordinate file. The latter is what is done in when running homme by itself (with file name passed via namelist); I pulled that detail in, but probably we don't want it, right?

@jgfouca tbc, it's ok to deal with those files like this, for now. I just want to understand if there are potentially other ascii file we are trying to copy here, cause, if not, I know we can remove this once we deal with vert coords directly from the IC netcdf file...

Reply:
@bartgol , I initially put this in to avoid copying the .nc files in `scream/data` because I was moving towards having SCREAM get them from DIN_LOC_ROOT instead. Now that the .nc files have been remove from our repo, this change is no longer necessary, so I will revert it.